pylsp/plugins/definition.py

```diff
@@ -8,26 +8,25 @@
 
 
 @hookimpl
-def pylsp_definitions(config, workspace, document, position):
-    with workspace.report_progress("go to definitions"):
-        settings = config.plugin_settings('jedi_definition')
-        code_position = _utils.position_to_jedi_linecolumn(document, position)
-        definitions = document.jedi_script(use_document_path=True).goto(
-            follow_imports=settings.get('follow_imports', True),
-            follow_builtin_imports=settings.get('follow_builtin_imports', True),
-            **code_position)
+def pylsp_definitions(config, document, position):
+    settings = config.plugin_settings('jedi_definition')
+    code_position = _utils.position_to_jedi_linecolumn(document, position)
```

Comment:
Did we decide to remove progress reporting from this one? Since this one is explicitly triggered by the user and can take long time, it feels like it should be OK to have it here IMO.

Reply:
> Did we decide to remove progress reporting from this one? Since this one is explicitly triggered by the user and can take longer time, it feels like it should be OK to have it here IMO.

Yes, we did. See [my comment](https://github.com/python-lsp/python-lsp-server/pull/328#issuecomment-1474940689) above for the reasons.

Reply:
> If that's wrong, I can change it back of course :)

Please don't. It's fine for now.

Reply:
Fine by me. BTW. I can't resolve issues created by myself so feel free to do that for me.